customization/settings.sqf

```diff
@@ -1,3 +1,13 @@
 //ace_cookoff_enable = false; //this disables cookoff fire effect
 //ace_cookoff_enableAmmoCookoff = false; //this disables secondary explosions
-ace_respawn_RemoveDeadBodiesDisconnected = false; //this disables automatically removing bodies
\ No newline at end of file
+ace_respawn_RemoveDeadBodiesDisconnected = false; //this disables automatically removing bodies
```

Comment:
How does this differ to corspeManagerMode and how do the two settings interact with each other?

Reply:
this was a setting from old olsen, so I am not sure. seems like leaving the false in there as default would be safe.

Reply:
Sounds good.